packages/postgres-database/src/simcore_postgres_database/migration/versions/65198ad31f29_quality_columns.py

```diff
@@ -0,0 +1,47 @@
+"""add quality to projects and services
+
+Revision ID: 65198ad31f29
+Revises: b60363fe438f
+Create Date: 2021-01-05 14:09:26.552123+00:00
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "65198ad31f29"
+down_revision = "b60363fe438f"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "projects",
+        sa.Column(
+            "quality",
+            postgresql.JSONB(astext_type=sa.Text()),
+            server_default=sa.text("'{}'::jsonb"),
```

Comment:
If you: open the Quality view -> press edit -> press Save button, the below objet will appear in the database. I would use that as a default for the migration as it is exactly what ends up being shown by the interface when first opening it on an empty study.

```json
    {
        "tsr": {
            "r01": {"level": 0, "references": ""},
            "r02": {"level": 0, "references": ""},
            "r03": {"level": 0, "references": ""},
            "r04": {"level": 0, "references": ""},
            "r05": {"level": 0, "references": ""},
            "r06": {"level": 0, "references": ""},
            "r07": {"level": 0, "references": ""},
            "r08": {"level": 0, "references": ""},
            "r09": {"level": 0, "references": ""},
            "r10": {"level": 0, "references": ""},
        },
        "annotations": {
            "vandv": "",
            "purpose": "",
            "standards": "",
            "limitations": "",
            "documentation": "",
            "certificationLink": "",
            "certificationStatus": "Uncertified",
        },
    }
```

Reply:
Yes that's the default fields that are added by the frontend. But it's not clear yet where this field should go and when it should be enabled I wouldn't add it to any migration script.